amundsen_application/static/js/components/DashboardPage/QueryListItem/styles.scss

```diff
@@ -0,0 +1,116 @@
+@import 'variables';
+
+$min-item-height: 54px;
+$query-content-padding: 12px;
+$query-content-border-radius: $spacer-1/2;
+$code-block-button-size: 32px;
+$code-block-button-radius: 4px;
+$code-block-button-icon-size: 16px;
+$code-block-button-spacer: 12px;
+$code-block-button-hover-opacity: 1;
+$code-block-max-height: 400px;
+
+
+@mixin button-style {
+  max-width: $code-block-button-size;
+  width: $code-block-button-size;
+  max-height: $code-block-button-size;
+  height: $code-block-button-size;
+  background: $body-bg;
+  border: 2px solid $stroke;
+  box-sizing: border-box;
+  border-radius: $code-block-button-radius;
+  color: $text-secondary;
+
+  svg {
+    fill: $text-secondary;
+  }
+
+  &:hover {
+    opacity: $code-block-button-hover-opacity;
+    text-decoration: none;
+  }
+}
+
+.query-list-header {
+  display: block;
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+  }
+}
+.query-list-item-name {
+  margin: 0;
+  line-height: $min-item-height;
+}
+.query-list-expanded-content {
+  padding-bottom: $spacer-3;
+}
+.query-list-query-label {
+  color: $text-tertiary;
+}
+.query-list-query-content {
+  border-radius: $query-content-border-radius;
+  border: 1px solid $stroke;
+  position: relative;
+  margin-top: $spacer-1/2;
+
+  > div {
+    padding: 0;
+
+    > span {
+      width: 100%;
+
+      > code {
+        width: 100%;
+        max-height: $code-block-max-height;
+        overflow-y: scroll;
+      }
+    }
+  }
+
+  button {
+    display: none;
+  }
+
+  &:hover {
+    button,
+    .query-list-query-link {
+      display: block;
+    }
+  }
+
+  svg.icon {
+    width: $code-block-button-icon-size;
+    height: $code-block-button-icon-size;
+  }
+
+  button {
+    @include button-style;
+    padding: 6px;
+    margin: $code-block-button-spacer (2 * $code-block-button-spacer) 0 0;
+  }
+
+  code {
+    padding: $query-content-padding !important;
+  }
```

Comment:
These are mainly overrides on the default styles of the third party component, hence the hackyness of the style definitions.